Formula/libpsl.rb

```diff
@@ -14,9 +14,12 @@ class Libpsl < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "python@3.8" => :build
   depends_on "libidn2"
 
   def install
+    ENV.prepend_path "PATH", Formula["python@3.8"].opt_libexec/"bin"
```

Comment:
Doesn't this happen automatically in superenv?

Reply:
Because `python@3.8` is keg_only I guess ...